contentcuration/contentcuration/utils/nodes.py

```diff
@@ -453,3 +453,13 @@ def migrate_extra_fields(extra_fields):
             "model": completion_criteria.MASTERY,
         }
     return extra_fields
+
+
+def validate_and_conform_to_schema_threshold_none(validated_options):
```

Comment:
My only suggestion would be to have this take the completion criteria, instead of the options object.